gulpfile.js

```diff
@@ -170,7 +171,7 @@ gulp.task('ractive-build-decorators', function () {
 	return gulp.src([
 		'./src/decorators/**/*.js',
 		'!./src/decorators/**/*.steps.js'
-	])
+		])
```

Comment:
Either the two lines above should be indented or this line shouldn't and the 3 lines below unindented.